hbase_sql/hbase1.1.1/src/main/java/com/splicemachine/hbase/BackupEndpointObserver.java

```diff
@@ -271,11 +282,12 @@ public void postFlush(ObserverContext<RegionCoprocessorEnvironment> e, Store sto
         try {
             // Register HFiles for incremental backup
             String filePath =  resultFile != null?resultFile.getFileInfo().getFileStatus().getPath().toString():null;
-            SpliceLogUtils.info(LOG, "Flushing store file %s", filePath);
+            SpliceLogUtils.info(LOG, "Flushed store file %s", filePath);
             if (!BackupUtils.isSpliceTable(namespace, tableName))
                 return;
             BackupUtils.captureIncrementalChanges(conf, region, fs, rootDir, backupDir,
                     tableName, resultFile.getPath().getName(), preparing.get());
+            isFlushing.set(false);
```

Comment:
We already set isFlushing to false in 

    public void postFlush(ObserverContext<RegionCoprocessorEnvironment> e) throws IOException {

why is it needed to do it here too?